airflow/dags/acs_2010_population.py

```diff
@@ -28,15 +28,35 @@
 acs_2010_bq_op = util.create_bq_ingest_operator(
     'acs_2010_gcs_to_bq', acs_2010_bq_payload, data_ingestion_dag)
 
-acs_2010_aggregator_payload = {'dataset_name': _ACS_2010_POPULATION_DATASET}
-acs_2010_aggregator_operator = util.create_aggregator_operator(
-    'acs_2010_aggregator', acs_2010_aggregator_payload,
+acs_2010_exporter_payload_race = {
+    'dataset_name': _ACS_2010_POPULATION_DATASET,
+    'demo_breakdown': "race"
```

Comment:
i have been avoiding `demographic` because to me that seems like it'd expect the value to be `black` or `male`, e/g/ the actual demographic. in this case it's the type of demographic disaggregation so it'd used `demo_breakdown` but i agree we should standardize. open to whatever but i think better to reserve `demographic` for the actual groups?

Reply:
switched to "demographic" since it's easier to make it work. but still don't love conflating the groups themselves with the method of breakdown

Reply:
That makes sense. I'm open to doing a mass renaming.